docs/example-scenario/ai/movie-recommendations.md

```diff
@@ -0,0 +1,117 @@
+* Consumer product recommendations in a mobile app.
+* News recommendations on streaming media.
+
+## Architecture
+
+![Architecture of a machine learning model for training movie recommendations][architecture]
+
+This scenario covers the training and evaluating of the machine learning model by using the Spark [alternating least squares][als] (ALS) algorithm on a dataset with ratings of movies. The steps for this scenario are as following:
+
+1. The front-end website or app service collects historical user-movie interaction data, which are usually represented in a table of user, item, and numerical rating tuples.
+2. The collected historical data is stored in a blob storage.
+3. A DSVM is often used to experiment with or productize a Spark ALS recommender model. The ALS model is trained using a training dataset, which is produced from the overall dataset by applying the appropriate data splitting strategy. For example, the dataset can be split into sets randomly, chronologically, or stratified, depending on the business requirement. Similar to other machine learning tasks, a recommender is validated by using evaluation metrics (for example, precision\@*k*, recall\@*k*, [MAP][map], [nDCG\@k][ndcg]).
+4. Azure Machine Learning service is used for coordinating the experimentation, such as hyperparameter sweeping and model management.
+5. A trained model is preserved on Azure Cosmos DB, which can then be applied for recommending the top *k* movies for a given user.
+6. The model is then deployed onto a web or app service by using Azure Container Instances or Azure Kubernetes Service.
+
+### Components
+
+* [Data Science Virtual Machine][dsvm] (DSVM) is an Azure virtual machine with deep learning frameworks and tools for machine learning and data science. The DSVM has a standalone Spark environment that can be used to run ALS.
+* [Azure Blob storage][blob] stores the dataset for movie recommendations.
+* [Azure Machine Learning service][mls] is used to accelerate the building, managing, and deploying of machine learning models.
+* [Azure Cosmos DB][cosmosdb] enables globally distributed and multi-model database storage.
+* [Azure Container Instances][aci](https://docs.microsoft.com/en-us/azure/container-instances/container-instances-overview) is used to deploy the trained models to web or app services, optionally using [Azure Kubernetes Service][aks].
+
+### Alternatives
+
+[Azure Databricks][databricks] is a managed Spark cluster where model training and evaluating is performed. You can set up a managed Spark environment in minutes, and [autoscale][autoscale] up and down to help reduce the resources and costs associated with scaling clusters manually. Another resource-saving option is to configure inactive [clusters][clusters] to terminate automatically.
+
+## Considerations
+
+### Availability
+
+The DSVM is available in [multiple regions][regions]  around the globe and meets the [service level agreement][sla] (SLA) for virtual machines.
```

Comment:
Let's talk about the different availability needs between training (not production critical, doesn't need to be built to be HA) and hosting (often does need to be built for HA)